src/redux/actions/bulkConcepts/index.js

```diff
@@ -29,3 +29,19 @@ export const addExistingBulkConcepts = data => async (dispatch) => {
   notify.show(`Added ${justAddedConcepts} concept(s) skipped ${alreadyAddedConcepts} that were already added`, 'success', 3000);
 };
 
+export const addDictionaryReference = (conceptUrl, ownerUrl, dictionaryId) =>
+  async (dispatch) => {
+    const url = `${ownerUrl}collections/${dictionaryId}/references/`;
+    let payload;
+    try {
+      const conceptsResponse = await instance.get(conceptUrl);
+      const references = {
+        data: { expressions: [...conceptsResponse.data.map(concept => concept.url)] },
+      };
+      payload = await instance.put(url, references);
+    } catch (error) {
+      notify.show('There was an error in referencing concepts in the new dictionary', 'error', 3000);
+    }
+    return dispatch(isSuccess(payload, ADD_EXISTING_BULK_CONCEPTS));
```

Comment:
Why does this function return a dispatch of `isSuccess` irrespective of whether there is an error or not?

Reply:
There suppose to be a `return` statement  in the catch scope, I'll make the update 👍.

Reply:
I have made the update